docs/contributing/index.rst

```diff
@@ -1,22 +1,35 @@
 Contributing
 ============
 
-.. toctree::
+* `Contributing to RestrictedPython`_
+
+  * `Legal requirements to contribute to RestrictedPython`_
+  * `Preperations for Contributing`_
+  * `Preperations for a new Python version`_
+  * `Enable a Python Feature in RestrictedPython`_
+
+* `Understanding How RestrictedPython works internally`_
+* `Differences between different Python versions`_
+
+.. _Contributing to RestrictedPython::
```

Comment:
Check the syntax for cross-referencing arbitrary locations: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role. I don't think whitespace is allowed in the reference name. If not, then the others should be updated as well.

```suggestion
.. _Contributing-to-RestrictedPython:
```